frontend/src/components/VCloseButton.vue

```diff
@@ -62,7 +62,7 @@ export default defineComponent({
      * The size of the underlying VIconButton.
      */
     size: {
-      type: String as PropType<"tiny" | "small" | "medium" | "large">,
+      type: String as PropType<"close" | "small" | "medium" | "large">,
```

Comment:
Can we bikeshed on the name?

Reply:
I would love to! Can you propose a better name? What can come between "tiny" and "small"?